frontend/pages/queries/ManageQueriesPage/ManageQueriesPage.tsx

```diff
@@ -146,24 +148,25 @@ const ManageQueriesPage = (): JSX.Element => {
     );
 
     try {
-      await Promise.all(removeQueries);
-      renderFlash("success", `Successfully removed ${queryOrQueries}.`);
-      toggleRemoveQueryModal();
-      refetchFleetQueries();
-      toggleRemoveQueryModal();
+      await Promise.all(removeQueries).then(() => {
+        dispatch(
+          renderFlash("success", `Successfully removed ${queryOrQueries}.`)
+        );
+        setResetSelectedRows(true);
+        refetchFleetQueries();
+      });
       dispatch(
         renderFlash("success", `Successfully removed ${queryOrQueries}.`)
       );
-      // TODO: Delete this redux action after redux dependencies have been removed (e.g., schedules page
```

Comment:
@RachelElysia please confirm that we are good to remove this based on our discussion today. Thanks!